specification/security/query_api_keys/types.ts

```diff
@@ -125,3 +144,88 @@ export type APIKeyAggregate =
   | RangeAggregate
   | DateRangeAggregate
   | CompositeAggregate
+
+export class APIKeyQueryContainer {
```

Comment:
```suggestion
/**
 * @variants container
 * @non_exhaustive
 */
export class APIKeyQueryContainer {
```

This needs to be decorated properly to be a container. :christmas_tree: